packages/slate-tables/src/TableEditor.tsx

```diff
@@ -1,50 +0,0 @@
-import type { BaseEditor, Element, Location } from 'slate';
-
-import * as TableCommands from './commands';
-import { TableNode } from './nodes';
-import * as TableQueries from './queries';
-
-export interface TableNodeType {
-    table: string;
-    row: string;
-    cell: string;
-}
-
-export interface TableSchema {
-    tableNodeTypes: TableNodeType;
```

Comment:
Replaced this with a set of `is{Type}Node` and `create{Type}Node` methods -- to not require the consumer app model having `type` property on nodes (same principle as with `slate-lists`).